core/arch/x86/decode_table.c

```diff
@@ -1498,6 +1498,7 @@ const instr_info_t * const op_instr[] =
 #define Wed TYPE_W, OPSZ_16_vex32_evex64
 #define Vex TYPE_V, OPSZ_16_vex32_evex64
 #define Wex TYPE_W, OPSZ_16_vex32_evex64
```

Comment:
These all look wrong too: 'e' means 'dq, qq, or oq', i.e., OPSZ_16_vex32_evex64.  Why is there another letter after the 'e' on these?

Reply:
We came up with this offline during another review. https://github.com/DynamoRIO/dynamorio/compare/91d1d006ed2f...58da5203eb41
AFAI can tell Intel uses Vs, Vd, Vx for type 'V' and single precision, double precision and 'x' for don't care. We enhanced it with 'e' for the invented evex version of 'V'. If you want Ve, how do you distinguish between Vs, Vd and Vx then for evex?

Reply:
I have never seen `Vs` or `Vd`.  Do you mean `vs` rather than `Vs`?  They are quite different.  But then there is no such things as `vx`.  See my comment above about {p,s,v,e}{s,d,i}.

Reply:
> We came up with this offline during another review

Yes the `ed` and `es` make sense for double and single float in a size `e` vector.  But the `ex` should just be `x`, just like the `dq` and other full-size-only codes that don't seem to care what the component types are.

Reply:
Whatever the name is, we still need to distinguish TYPE_W, OPSZ_16_vex32_evex64 and TYPE_W, OPSZ_16_vex32. I can name the latter Wx, and the former We?

Reply:
Yes, that is what they would be, We and Wx.

Reply:
Done.